app/main/views.py

```diff
@@ -58,19 +58,22 @@ def find():
 
 @main.route('/service/<service_id>')
 def view(service_id):
-    service_data = data_api_client.get_service(service_id)
-    presented_service_data = {}
-    for key, value in service_data.items():
-        presented_service_data[key] = presenters.present(
-            value, content.get_question(key)
-        )
-
-    template_data = get_template_data({
-        "sections": content.sections,
-        "service_data": presented_service_data,
-        "service_id": service_id
-    })
-    return render_template("view_service.html", **template_data)
+    try:
+        service_data = data_api_client.get_service(service_id)
+        presented_service_data = {}
+        for key, value in service_data.items():
+            presented_service_data[key] = presenters.present(
+                value, content.get_question(key)
+            )
+
+        template_data = get_template_data({
+            "sections": content.sections,
+            "service_data": presented_service_data,
+            "service_id": service_id
+        })
+        return render_template("view_service.html", **template_data)
+    except APIError as e:
```

Comment:
Unless I'm missing something I think we can just wrap the `service_data = data_api_client.get_service(service_id)` line